### DISCUSSION TITLE
HandshakeException on Client Login

### DISCUSSION THREAD

**Discussion by nercox** [rustdesk/rustdesk-server-pro#350] [Q&A]
Hi everyone,

I have installed RustDeskServer Pro on my Windows Server and generated clients. Connecting to the clients works fine. However, logging in from the client application does not work (it does work on the web). The error shown in the image appears:
![Screenshot 2024-08-07 180658](https://github.com/user-attachments/assets/6999a679-69f7-4487-bc31-ec8854b005cb)

need help. thx

**Comment by nercox** [comment]
I also can't find any entry in the log about it, and I have directly included a certificate in the configuration.
![Screenshot 2024-08-07 180838](https://github.com/user-attachments/assets/4f4c123e-98c0-4903-92f6-08795cd9c770)

**Reply by rustdesk** [reply]
We do not have exp of setting up https on Windows with cert like this, really hard to help you.
